corona-warn-app/cwa-app-android
# Fix countries missing in onboarding after in-app reset (EXPOSUREAPP-4203)

Remove the extra flow that would cache the countries. The app config itself is cached already.
When accessing the country list they will be mapped from the latest available app config.

### Testing
* Check the country list in on-boarding and interop settings
* Reset the app via in-app data reset
* Check the country list again
